week4/search.py

```diff
@@ -138,7 +157,15 @@ def query():
     query_class_model = current_app.config["query_model"]
     query_category = get_query_category(user_query, query_class_model)
     if query_category is not None:
-        print("IMPLEMENT ME: add this into the filters object so that it gets applied at search time.  This should look like your `term` filter from week 1 for department but for categories instead")
+        terms_fitler = {
+            "terms": {
+                "categoryPathIds": query_category
```

Comment:
Unsure about this field, but there is `categoryPathIds.keyword` that seems more suitable as a keyword field.

Reply:
Yeah. Actually I was using my First week index here, There categoryPathIds was mapped  as a keyword field, text field was not created for this. 

```
      "categoryPathIds": {
        "type": "keyword"
      },
```

Reply:
Ok!